src/workstation-gui/Gemfile

```diff
@@ -0,0 +1,82 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# ruby "3.0.0"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.0.3"
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem "sprockets-rails"
+
+# gem "tzinfo-data"
+# gem 'concurrent-ruby', '~> 1.1', '>= 1.1.10'
+
+# Use sqlite3 as the database for Active Record
+# gem "sqlite3", "~> 1.4"
+
+# Use the Puma web server [https://github.com/puma/puma]
+gem "puma", "~> 5.0"
+
+gem "net-smtp", require: false
+
+gem "premailer-rails", group: [:development, :production]
+
+# Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
+gem "importmap-rails"
+
+# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
+gem "turbo-rails"
+
+# Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
+gem "stimulus-rails"
+
+# Build JSON APIs with ease [https://github.com/rails/jbuilder]
+gem "jbuilder"
+
+# Use Redis adapter to run Action Cable in production
+# gem "redis", "~> 4.0"
+
+# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
+# gem "kredis"
+
+# Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
+# gem "bcrypt", "~> 3.1.7"
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
+
+# Reduces boot times through caching; required in config/boot.rb
+gem "bootsnap", require: false
+
+gem "nokogiri"
+
+# Use Sass to process CSS
+# gem "sassc-rails"
+
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
+
+# group :development, :test do
+#   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+#   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+# end
+
+# group :development do
+#   # Use console on exceptions pages [https://github.com/rails/web-console]
+#   gem "web-console"
+
+#   # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
+#   # gem "rack-mini-profiler"
+
+#   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
+#   # gem "spring"
+# end
```

Comment:
do we need to keep the commented out gems?

Reply:
No couple of them we will be needing but which are not been used I have removed those